solutions/security/ai/attack-discovery.md

```diff
@@ -54,12 +58,14 @@ Attack Discovery is designed for use with alerts based on data that complies wit
 The selected fields can now be analyzed the next time you run Attack Discovery.
 :::
 
-## Generate discoveries [attack-discovery-generate-discoveries]
+## Generate discoveries manually[attack-discovery-generate-discoveries]
```

Comment:
Github won't let me comment on the actual line above (because it's not part of this PR), but in a previous release the `Chat` button got renamed to `Ask AI Assistant` as illustrated by the screenshot below:

![ask_ai_assistant](https://github.com/user-attachments/assets/3f6d02e2-eb24-4b4b-80fa-916b70be3601)

Would you be willing to change `Chat` to `Ask AI Assistant` in the follwing statement (on line `54` above)?

```
1. Select an alert with some of the non-ECS fields you want to analyze, and go to its details flyout. From here, use the **Chat** button to open AI Assistant.
```